Makefile

```diff
@@ -105,6 +105,7 @@ go_get_agent_deps: go_get_common_deps go_get_ping_deps
 	$(GOGETTER) github.com/jvehent/service-go
 	$(GOGETTER) camlistore.org/pkg/misc/gpgagent
 	$(GOGETTER) camlistore.org/pkg/misc/pinentry
+	$(GOGETTER) github.com/ameihm0912/mozoval/go/src/oval
```

Comment:
it's a detail, but since we're going to make production use of mozoval, what about hosting it under `mozilla/mozoval`?